### PR TITLE
Expense service tests

### PR DESCRIPTION
**Pull request by Stanislavcod**

### CODE REVIEW

File: ExpenseMaster.Tests/Repository/BudgetRepositoryTests.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            };
+            var mockDbSet = GetMockDbSet(budgets);
+            var mockContext = new Mock<ApplicationDatabaseContext>();
+            mockContext.Setup(c => c.Set<Budget>()).Returns(mockDbSet.Object);
+
+            var repository = new BudgetRepository(mockContext.Object);
+
+            var result = await repository.GetByIdAsync(2);
+
+            Assert.NotNull(result);
```
Comment (KatsiarynaAnashkevich) [inline ExpenseMaster.Tests/Repository/BudgetRepositoryTests.cs:29]: As i remember we talk about fluent assertion library

File: ExpenseMaster.Tests/Repository/BudgetRepositoryTests.cs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using Xunit;
+
+namespace ExpenseMaster.Tests.Repository
+{
+    public class BudgetRepositoryTests
+    {
+        [Fact]
+        public async Task GetByIdAsync_ReturnsBudget()
```
Comment (KatsiarynaAnashkevich) [inline ExpenseMaster.Tests/Repository/BudgetRepositoryTests.cs:13]: And we talk about three parts in name of test

File: ExpenseMaster.Tests/Services/BudgetServiceTests.cs [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+            var mockMapper = new Mock<IMapper>();
+
+            mockRepositoryWrapper
+                .Setup(r => r.Budget.GetBudgetsByUserIdAsync(userId))
+                .ReturnsAsync((List<Budget>)null);
+
+            var budgetService = new BudgetService(mockRepositoryWrapper.Object, mockMapper.Object);
+
+            // Act and Assert
+            await Assert.ThrowsAsync<InvalidOperationException>(() => budgetService.GetByUserIdAsync(userId));
```
Comment (KatsiarynaAnashkevich) [inline ExpenseMaster.Tests/Services/BudgetServiceTests.cs:201]: separate them

File: ExpenseMaster.Tests/Repository/BudgetRepositoryTests.cs (hunk 2 above)
Comment (Stanislavcod) [reply ExpenseMaster.Tests/Repository/BudgetRepositoryTests.cs:13]: it's not my code and it's old

File: ExpenseMaster.Tests/Services/BudgetServiceTests.cs (hunk 3 above)
Comment (Stanislavcod) [reply ExpenseMaster.Tests/Services/BudgetServiceTests.cs:201]: it's not my code and it's old

**Pull request merged**